File: minijeu.py
import random

def main():
    nombre_mystere = random.randint(1, 100)
    essais = 0
    
    print("=== Devinez le nombre mystère (entre 1 et 100)  ===")
    
    while True:
        nombre_joueur = int(input("Entrez un nombre : "))
        essais += 1
        
        if nombre_joueur < nombre_mystere:
            print("C'est plus !")
        elif nombre_joueur > nombre_mystere:
            print("C'est moins !")
        else:
            print(f"Bravo, vous avez trouvé le nombre mystère en {essais} essais !")
            break

if __name__ == "__main__":
    main()

File: minijeu.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    int nombreMystere, nombreJoueur, essais = 0;
    srand(time(NULL));
    nombreMystere = rand() % 100 + 1;  // Génère un nombre aléatoire entre 1 et 100
    
    printf("=== Devinez le nombre mystère (entre 1 et 100) ===\n");
    
    do {
        printf("Entrez un nombre : ");
        scanf("%d", &nombreJoueur);
        essais++;
        
        if (nombreJoueur < nombreMystere) {
            printf("C'est plus !\n");
        } else if (nombreJoueur > nombreMystere) {
            printf("C'est moins !\n");
        } else {
            printf("Bravo, vous avez trouvé le nombre mystère en %d essais !\n", essais);
        }
        
    } while (nombreJoueur != nombreMystere);
    
    return 0;
}
